src/clients/utils.ts

```diff
@@ -33,3 +33,9 @@ export async function inBatches<T, K>(
 export function trimLastForwardSlash(url: string) {
   return url.replace(/\/$/, '')
 }
+
+export function isLocalLink(href?: string | null) {
+  return (
+    typeof href === 'string' && !href.startsWith('https://') && !href.startsWith('http://') && !href.startsWith('//')
```

Comment:
I think we could write this like:

```typescript
return typeof href === 'string' && !/^((https?):)?\/\//.test(href)
```


![image](https://github.com/decentraland/governance/assets/2858950/2bb6b358-0e57-4293-83d8-f107964cbcf3)